### DISCUSSION TITLE
req: Setup discord for participants

### DISCUSSION THREAD

**Discussion by tormentr** [Cyfrin/foundry-full-course-cu#49] [Q&A] [answered]
Repo Discussions are very helpful no doubt , If possible an official discord could be setup for discussions,

**Comment by cromewar** [comment]
@PatrickAlphaC I can setup this if you think this could be a good idea.

**Reply by PatrickAlphaC** [reply, marked as answer]
@tormentr I'd rather have all discussions for all repos here!
